### DISCUSSION TITLE
[Mac] Markdown Pro editor renders inline code too large and overlaps surrounding lines

### DISCUSSION THREAD

**Discussion by austen0** [standardnotes/forum#2479] [General] [answered]
**Describe the bug**
Markdown Pro editor renders inline code too large and overlaps surrounding lines.

**To Reproduce**
Write a multi-line paragraph or list and format some text as code, then preview.

**Expected behavior**
Things aren't rendered over one another.

**Screenshots**
![Screen Shot 2021-11-30 at 9 21 57 AM](https://user-images.githubusercontent.com/17052885/144096670-6c02b61d-e700-435f-889a-b58564d772ed.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Mac
 - Browser [e.g. chrome, safari] N/A
 - Version [e.g. 22] 3.9.5

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] N/A
 - OS: [e.g. iOS8.1] N/A
 - Browser [e.g. stock browser, safari] N/A
 - Version [e.g. 22] N/A

**Additional context**
N/A

**Comment by myreli** [comment, marked as answer]
It seems to be resolved as of `v3.148.0`

<img width="743" alt="image" src="https://user-images.githubusercontent.com/17554234/218531340-c5666618-7717-4147-af7c-0218700595e0.png">
